projects/Benchmarks/Networking/RabbitMqBroker.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading;
+using Ductus.FluentDocker.Builders;
+
+namespace Benchmarks.Networking
+{
+    public class RabbitMqBroker
```

Comment:
Can we make this say `RabbitMQ` and not `RabbitMq`?

Reply:
Or even `RabbitMQNode`.

Reply:
sure